### PR TITLE
Route Views Code of conduct

### PR DESCRIPTION
**Pull request by ryeleo**
I am considering that Route Views might want to try following the: 'Contributor Covenant "Code of Conduct" v2.1'

### Review Instructions

Take a look through the [code of conduct (rendered)](https://github.com/routeviews/issues/blob/ea3778796ae59baec1194094cb9c6148cd4fc5af/docs/code-of-conduct.md) suggested in this pull request.

I believe this is something that we'll want consensus on by all @routeviews/maintainers. Please feel free to discuss this code of conduct in this pull request or via Slack! 

If you agree that we should take on this code of conduct for now, please check the box below associated with your name, or approve this pull request:

- [ ] @hanskuhn approves
- [x] @kemp-rv approves
- [x] @pokui approves
- [ ] @pfsinoz approves
- [x] @ryeleo approves

> Not requiring confirmation by currently inactive maintainers: @hervey and @kconte 

> ℹ Extra motivation: One benefit of getting this PR complete -- we won't have a big TODO in our [issue-forms](https://github.com/routeviews/issues/issues/new/choose) anymore:
> 
image

### CODE REVIEW

File: docs/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, or sexual
+identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
```
Comment (ryeleo) [inline docs/code-of-conduct.md:17]: I *really* appreciate the examples discussed in the "Our Standards" section -- I think that is the most useful part of this document for the general contributors who will be glancing over this (when they submit an issue).

**Comment by ryeleo** [comment]
Hey y'all, reviving this PR with a nudge to our primary RV Maintainers, who I believe will appreciate having a 'code of conduct' for Route Views. 🤓👍 
Specifically nudging: @hanskuhn, @pokui, @dteach-rv, @pfsinoz

File: docs/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+communication with the community for a specified period of time. No public or
+private interaction with the people involved, including unsolicited interaction
+with those enforcing the Code of Conduct, is allowed during this period.
+Violating these terms may lead to a permanent ban.
+
+### 4. Permanent Ban
+
+**Community Impact**: Demonstrating a pattern of violation of community
+standards, including sustained inappropriate behavior, harassment of an
+individual, or aggression toward or disparagement of classes of individuals.
```
Comment (pokui) [inline docs/code-of-conduct.md:110]: much as we don't expect this I think it's an important clause to have

**Review by pokui — APPROVED** [review]

File: docs/code-of-conduct.md (hunk 2 above)
Comment (ryeleo) [reply docs/code-of-conduct.md:110]: I've unfortunately heard that some past RV collaborators used disrespectful words that discriminated against some groups -- thankfully I haven't run into it yet. I really like that this code of conduct gives us a clear route forward if we do find ourselves working with unsavory folks!

**Pull request closed without merging**